docs/source/cpp/compute.rst

```diff
@@ -140,8 +140,12 @@ Aggregations
 +--------------------------+------------+--------------------+-----------------------+--------------------------------------------+
 | mode                     | Unary      | Numeric            | Scalar Struct  (2)    |                                            |
 +--------------------------+------------+--------------------+-----------------------+--------------------------------------------+
+| std                      | Unary      | Numeric            | Scalar Float64        | :struct:`VarStdOptions`                    |
++--------------------------+------------+--------------------+-----------------------+--------------------------------------------+
 | sum                      | Unary      | Numeric            | Scalar Numeric (3)    |                                            |
 +--------------------------+------------+--------------------+-----------------------+--------------------------------------------+
+| var                      | Unary      | Numeric            | Scalar Float64        | :struct:`VarStdOptions`                    |
```

Comment:
Would rather "stddev" and "variance" respectively. "std" and "var" can easily be misunderstood, IMHO.

Reply:
done